funnel/views/account.py

```diff
@@ -173,6 +174,25 @@ def recent_organization_memberships(
     )
 
 
+@Account.views()
+def organizations_as_member(
+    obj: Account, limit: int | None = None, order_by_grant: bool = False
+) -> list[RoleAccessProxy]:
+    """Return organizations that the user has a membership in"""
+    featured_accounts = Account.query.filter(
+        Account.name_in(app.config['FEATURED_ACCOUNTS'])
+    ).all()
+    org_as_members = []
+    for org in featured_accounts:
+        membership_project = org.projects.filter(
+            Project.boxoffice_data.op('@>')({'has_membership': True})
+        ).first()
+        print('membership_project', membership_project)
+        if membership_project and membership_project.current_roles.account_member:
```

Comment:
You don't need this, as `account_member` is granted from the account, and it finds the relevant membership project.

You can remove the project query and simply check for `org.current_roles.member`.